src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -157,4 +157,44 @@
   <Import Project="$(BuildCommonDirectory)common.targets" />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != '' and '$(DotNetBuildFromSource)' != 'true'" />
+
+  <Target Name="DetermineGenerateProductVersionAssemblyInfoOutputs">
+    <ItemGroup>
+      <_ProductVersionAssemblyInfoFile Include="$(IntermediateOutputPath)ProductVersion.g$(DefaultLanguageSourceExtension)">
+        <Source>
+<![CDATA[
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by $(MSBuildThisFileFullPath).
+//
+//     Changes to this file may cause incorrect behavior and will be lost if the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace $(RootNamespace)
+{
+    public sealed partial class NuGetPackage
+    {
+        /// <summary>
+        /// The current version of the NuGet Visual Studio package which is displayed in the Help - About box of Visual Studio
+        /// </summary>
+        public const string ProductVersion = "$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)"%3B
```

Comment:
You can probably just say `SemanticVersion`.

Reply:
Good idea, I also just updated the comment in `config.props`.